test/cases/ostree-ng.sh

```diff
@@ -84,7 +83,9 @@ COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
 
 # SSH setup.
 SSH_OPTIONS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5)
-SSH_KEY=${OSBUILD_COMPOSER_TEST_DATA}keyring/id_rsa
+SSH_DATA=$(/usr/libexec/osbuild-composer-test/gen-ssh.sh)
```

Comment:
This should ideally be `SSH_DATA_DIR` but I'm not strongly opposed to the current name.